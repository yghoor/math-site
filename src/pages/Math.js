/* eslint-disable no-unused-vars */
import Calculator from '../components/Calculator.js';

const Math = () => (
    <div className="math-page">
      <h1>Let's do some math!</h1>
      <Calculator />
    </div>

);

export default Math;