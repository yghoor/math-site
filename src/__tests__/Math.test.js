/* eslint-disable no-unused-vars */
import renderer from 'react-test-renderer';

import Math from '../pages/Math.js';

test('renders "Math" page', () => {
  const tree = renderer
    .create(<Math />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});