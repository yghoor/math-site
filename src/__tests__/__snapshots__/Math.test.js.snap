// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Math" page 1`] = `
<div
  className="math-page"
>
  <h1>
    Let's do some math!
  </h1>
  <section
    className="calculator"
  >
    <div
      className="calculator-display"
    >
      <div
        className="display-value"
      >
        0
      </div>
    </div>
    <div
      className="buttons"
    >
      <button
        className="button value-button"
        onClick={[Function]}
      >
        AC
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        +/-
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        %
      </button>
      <button
        className="button operator-button"
        onClick={[Function]}
      >
        รท
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        7
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        8
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        9
      </button>
      <button
        className="button operator-button"
        onClick={[Function]}
      >
        x
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        4
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        5
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        6
      </button>
      <button
        className="button operator-button"
        onClick={[Function]}
      >
        -
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        1
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        2
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        3
      </button>
      <button
        className="button operator-button"
        onClick={[Function]}
      >
        +
      </button>
      <button
        className="button value-button zero-button"
        onClick={[Function]}
      >
        0
      </button>
      <button
        className="button value-button"
        onClick={[Function]}
      >
        .
      </button>
      <button
        className="button operator-button"
        onClick={[Function]}
      >
        =
      </button>
    </div>
  </section>
</div>
`;
